lib/sql-manager/index.js

```diff
@@ -117,6 +112,52 @@ try {
         default: throw Error('messed up query')
       }
     }
+
+    createTable ({ name, columns }) {
+      return this._qb.schema.hasTable(name)
+        .then((exists) => {
+          if (exists) {
+            throw Error(`Table with name ${name} already exists.`)
+          }
+          return this._qb.schema.createTable(name, (table) => {
+            if (this._qb.client.config.client === 'mysql') {
+              table.charset('utf8')
+            }
+            for (const column of columns) {
+              if (!table[column.type]) {
+                throw Error(
+                  `'${column.name}' uses '${column.type}'
+                which is not an existing type.`
+                )
+              }
+              if (column.primary === true && column.default !== undefined) {
+                table[column.type](column.name)
+                  .primary()
+                  .defaultTo(column.default)
+              } else if (column.primary === true) {
+                table[column.type](column.name)
+                  .primary()
+              } else if (column.default !== undefined) {
+                table[column.type](column.name)
+                  .defaultTo(column.default)
+              } else {
+                table[column.type](column.name)
+              }
+            }
+          })
+            .then(() => {
+              return {
+                name,
+                columns
+              }
+            })
+        })
+    }
+
+    dropTable (table) {
```

Comment:
If the table exists, return the table name. If it doesn't, return `undefined`

Reply:
this has been removed